frontend/next-env.d.ts

```diff
@@ -1,5 +1,4 @@
 /// <reference types="next" />
-/// <reference types="next/types/global" />
```

Comment:
See above, Are we sure we don't need this?

Reply:
This was removed when I ran `yarn run build` locally for testing purposes – everything runs fine without it, but I'll add it back just in case.